packages/web/config/user-analytics-v2.ts

```diff
@@ -0,0 +1,135 @@
+    swapCompleted: "Swap: Swap completed",
+  },
+  // Events in Sidebar UI
+  Sidebar: {
+    stakeClicked: "Sidebar: Stake clicked",
+    voteClicked: "Sidebar: Vote clicked",
+    infoClicked: "Sidebar: Info clicked",
+    connectWalletClicked: "Sidebar: Connect wallet clicked",
+    signOutClicked: "Sidebar: Sign out clicked",
+    twitterClicked: "Sidebar: Twitter clicked",
+    mediumClicked: "Sidebar: Medium clicked",
+    commonwealthClicked: "Sidebar: Commonwealth clicked",
+    discordClicked: "Sidebar: Discord clicked",
+    telegramClicked: "Sidebar: Telegram clicked",
+    supportClicked: "Sidebar: Support clicked",
+  },
+  // Events in Pools page
+  Pools: {
+    pageViewed: "Pools: Page viewed",
+    createNewPoolClicked: "Pools: Create new pool clicked",
+    myPoolsCardClicked: "Pools: My pools card clicked",
+    superfluidPoolsCardClicked: "Pools: Superfluid pools card clicked",
+    allPoolsListFiltered: "Pools: All pools list filtered",
+    allPoolsListSorted: "Pools: All pools list sorted",
+    allPoolsTableSorted: "Pools All pools table sorted",
+    incentivizedPoolsItemClicked: "Pools: Incentivized pools item clicked",
+    allPoolsItemClicked: "Pools: All pools item clicked",
+    externalIncentivePoolsListSorted:
+      "Pools: External incentive pools item sorted",
+    externalIncentivePoolsTableSorted:
+      "Pools: External incentive pools table sorted",
+    externalIncentivePoolsItemClicked:
+      "Pools: External incentive pools item clicked",
+  },
+  // Events in Pool detail page
+  PoolDetail: {
+    pageViewed: "Pool detail: Page viewed",
+    addOrRemoveLiquidityClicked: "Pool detail: Add/Remove liquidity clicked",
+    swapTokensClicked: "Pool detail: Swap tokens clicked",
+    startEarningClicked: "Pool detail: Start earning clicked",
+    unbondAllStarted: "Pool detail: Unbond all started",
+    unbondAllCompleted: "Pool detail: Unbond all completed",
+    singleAssetClicked: "Manage liquidity: Single asset clicked",
+    addLiquidityStarted: "Manage liquidity: Add liquidity started",
+    addLiquidityCompleted: "Manage liquidity: Add liquidity completed",
+    removeLiquidityStarted: "Manage liquidity: Remove liquidity started",
+    removeLiquidityCompleted: "Manage liquidity: Remove liquidity completed",
+    bondStarted: "Liquidity bonding: Bond started",
+    bondCompleted: "Liquidity bonding: Bond completed",
+    superfluidStakeStarted: "Liquidity bonding: Superfluid stake started",
+    superfluidStakeCompleted: "Liquidity bonding: Superfluid stake completed",
+  },
+  // Events in assets page
+  Assets: {
+    pageViewed: "Assets: Page viewed",
+    depositClicked: "Assets: Deposit clicked",
+    withdrawClicked: "Assets: Deposit clicked",
+    myPoolsCardClicked: "Assets: My pools card clicked",
+    myPoolsMoreClicked: "Assets: My pools more clicked",
+    assetsListFiltered: "Assets: Assets list filtered",
+    assetsListSorted: "Assets: Assets list sorted",
```

Comment:
Why do we have separate events for sorting assets? I think in general with events we should decouple them from the UI a bit, so I think it's more fitting to have something like "assets sorted" as a more general event.

Not important.

Reply:
I think you are right. I just wanted to figure out what sorting UI(in dropdown or table head) user using but It can be figured out by adding event property not separated events.